src/components/Web3ReactManager/index.tsx

```diff
@@ -21,12 +21,12 @@ export default function Web3ReactManager({ children }: { children: JSX.Element }
   const dispatch = useDispatch()
   /** On user change network from wallet, update chainId in store, only work on EVM wallet */
   useEffect(() => {
-    if (triedEager && chainIdEVM && chainIdState !== chainIdEVM && active && isEVM) {
+    if (triedEager.current && chainIdEVM && chainIdState !== chainIdEVM && active && isEVM) {
       dispatch(updateChainId(chainIdEVM))
     }
     // Only run on change network from wallet
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [chainIdEVM, triedEager, active])
+  }, [chainIdEVM, triedEager.current, active])
 
   // on page load, do nothing until we've tried to connect to the injected connector
   if (isEVM && !triedEager) {
```

Comment:
!triedEager. current

Reply:
this is dependency list. You sure?

Reply:
I mean this line:
 if (isEVM && !triedEager) {

Reply:
fixed